### PR TITLE
#168399940 Articles by Tags

### PR DESCRIPTION
**Pull request by Hadeneekeh**
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168399940](https://www.pivotaltracker.com/n/projects/2364267)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -22,3 +22,5 @@ NEW_USER_PASSWORD=default password of a new user
 FACEBOOK_SDK = 'string'
 TWITTER_SDK = 'string'
 FRONT_END_URL = 'string'
+SECRET_KEY_STRIPE = STRING - stripe secret key
+PUBLISHABLE_KEY_STRIPE=  STRING - stripe publishable key
```
Comment (Lundii) [inline .env.sample]: Nice work Kafilat. Can you add a new line to this file?

**Review by encodedBicoding — APPROVED** [review]
LGTM

**Review by obayomi96 — APPROVED** [review]
LGTM

**Review by chuxmykel — APPROVED** [review]
LGTM

**Review by cvjude — APPROVED** [review]
LGTM

**Review by chiboycalix — APPROVED** [review]
LGTM

**Pull request merged**